packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -1085,113 +1071,5 @@ describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider, com
 			ds2.context.id.length > 8,
 			"Data store's ID in attached container should not be short",
 		);
-		const dds2 = SharedDirectory.create(ds2.runtime);
-		assert(dds2.id.length > 8, "DDS's ID in attached container should not be short");
 	});
 });
-
-/**
- * These tests repro a bug where ODSP driver does not correctly decode encoded snapshot tree paths.
- * Data store / DDS created with special characters are encoded during summary upload but during
- * download, they are not correctly decoded in certain scenarios.
- */
-describeCompat(
-	"Short IDs in detached container",
-	"NoCompat",
-	(getTestObjectProvider, apis) => {
-		const configProvider = createTestConfigProvider();
-		const {
-			dataRuntime: { TestFluidObjectFactory },
-			containerRuntime: { ContainerRuntimeFactoryWithDefaultDataStore },
-			dds: { SharedDirectory },
-		} = apis;
-		const defaultFactory = new TestFluidObjectFactory([]);
-		const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore({
-			defaultFactory,
-			registryEntries: [[defaultFactory.type, Promise.resolve(defaultFactory)]],
-		});
-
-		let provider: ITestObjectProvider;
-
-		beforeEach("getTestObjectProvider", async function () {
-			provider = getTestObjectProvider();
-			// The bug only happens with ODSP driver.
-			if (provider.driver.type !== "odsp") {
-				this.skip();
-			}
-			configProvider.set("Fluid.Runtime.IsShortIdEnabled", true);
-		});
-
-		/**
-		 * The following functions validate the invert of the assert. We have a bug in one of our customer's app where a short
-		 * data store ID created is `[` but in a downloaded snapshot, it is converted to its ASCII equivalent `%5B` in
-		 * certain conditions. So, when an op comes for this data store with id `[`, containers loaded with this snapshot
-		 * cannot find the data store.
-		 *
-		 * While we figure out the fix, we are disabling the ability to create short IDs and this assert validates it.
-		 */
-		function assertInvert(value: boolean, message: string) {
-			assert(!value, message);
-		}
-
-		it("data store id with `[` not encoded / decoded correctly in snapshot`", async () => {
```

Comment:
Deleted the tests that depended on "[" id creation. It is no longer valid as now are now only using encode safe characters to generate ids.